src/java.base/share/classes/sun/security/util/ECParameters.java

```diff
@@ -213,7 +214,8 @@ protected void engineInit(byte[] params, String decodingMethod)
         }
 
         throw new InvalidParameterSpecException(
-            "Only ECParameterSpec and ECGenParameterSpec supported");
+            "Only ECParameterSpec, ECGenParameterSpec " +
+                    "and ECKeySizeParameterSpec supported");
```

Comment:
Would you mind reducing the leading whitespaces so that the two lines are aligned?